README.md

```diff
@@ -53,6 +54,32 @@ make start-db run
 
 This runs a server using the default virtual environment. Documentation can be found at  `http://127.0.0.1:8081/docs`.
 
+### Relevant APIs
+
+The `/relevant/` APIs query [host inventory] in order to return relevant results. To avoid querying the inventory API and return fixture data, set `ROADMAP_DEV=1` in the environment.
+
+```
+export ROADMAP_DEV=1
+make run
+```
+
+This will return data from [tests/fixtures/](tests/fixtures), making it easy to change the response for testing and development.
+
+### Getting a token for accessing Red Hat APIs
+
+In order to query host inventory, a Red Hat API access token is required. Access tokens are only valid for fifteen minutes and require an [offline token] in order to generate new ones.
+
+```
+export RH_OFFLINE_TOKEN="[offline token]"
+export RH_TOKEN="$(./scripts/get-redhat-access-token.py)"
+```
+
+Use the access token in the request header. Here is an example using [httpie].
+
+```
+http localhost:8081/api/roadmap/v1/relevant/lifecycle/rhel/ \
```

Comment:
This returns `404 Not found`, it's missing the commits from the branch with those changes. But I understand there is some "race" between merging those and relevant PR is still draft, so I don't see it as a problem now. I understand it as some preparation and when relevant get merged, it will work great.

(or we can use this for getting the token and try it in the relevant branch, I just think it's good to mention there :))

Reply:
Correct. The changes in #39 will require using a token but nothing currently merged needs it.